docs/modules/commons/pages/vividus-steps.adoc

```diff
@@ -138,3 +138,17 @@ Given I initialize the scenario variable `personalInfo` using template `template
 |name     |race  |
 |Dagoth Ur|Dunmer|
 ----
+
+=== Assert the value matches regex
+
+Asserts that the given value matches the specified regular expression.
+The dotall mode is enabled by default: the expression `.` matches any character,
+including a line terminator.
```

Comment:
should not we also add that dot all matches ^ and $ only at the start and end of string?

Reply:
`^` and `$` match start/end of line online in conjunction with MULTILINE flag